src/trapfunc.cpp

```diff
@@ -822,17 +822,23 @@ void trapfunc::lava(Creature *c, int x, int y)
 }
 
 // STUB
-void trapfunc::portal(Creature */*c*/, int /*x*/, int /*y*/)
-{
+void trapfunc::portal(
```

Comment:
You're throwing the coding style out the window here.  Plus I'm not sure what warnings this would be silencing.